markdown/3.x/en/documentation/attribute-basics.md

```diff
@@ -12,14 +12,15 @@ Every element can have attributes of various types. These types are `Boolean`, `
 For attributes of type `Boolean` only their presence on the HTML element is relevant. Their assigned value is ignored. The presence of a boolean attribute on an element represents the `true` value, and the absence of the attribute represents the `false` value.
 
 ```html
-<ff-communication search-immediate></ff-communication>
-<ff-communication search-immediate="false"></ff-communication>
+<ff-asn-group opened></ff-asn-group>
```

Comment:
This switch of elements was done because `opened` sounds more like a Boolean attribute or there's a different reason?

Reply:
I switched because `opened` is a real boolean attribute. `ff-communication` has a special implementation for the boolean attributes that make `search-immediate="false"` be interpreted as `false`. This special confused users, and in fact it is undocumented. So I figured `opened` is a better example.